src/frontend/js/widgets/Dashboard/components/DashboardAvatar/index.tsx

```diff
@@ -1,6 +1,32 @@
-import { User } from 'types/User';
+import c from 'classnames';
 
-export const DashboardAvatar = ({ user }: { user: User }) => {
-  const letter = user.username.charAt(0).toUpperCase();
-  return <div className="dashboard__avatar">{letter}</div>;
+export enum DashboardAvatarVariantEnum {
+  DEFAULT = 'default',
+  SQUARE = 'square',
+}
+
+interface DashboardAvatarProps {
+  title: string;
+  imageUrl?: string;
+  variant?: DashboardAvatarVariantEnum;
+}
+
+export const DashboardAvatar = ({
+  title,
+  imageUrl,
+  variant = DashboardAvatarVariantEnum.DEFAULT,
+}: DashboardAvatarProps) => {
+  const letter = title.charAt(0).toUpperCase();
+  return (
+    <div
+      data-testid="dashboard-avatar"
+      className={c('dashboard-sidebar__container__header__avatar', {
+        dashboard__avatar__square: variant === DashboardAvatarVariantEnum.SQUARE,
```

Comment:
It appears more logic to me to apply this class to `dasbhoard__avatar` element ?

Reply:
This class moved there when i realised i need it to position the container.
* circle avatar get 1/2 of they height higher than the top of the sidebar.
* square avatar get 3/4 of they height higher than the top of the sidebar.

Reply:
But I feel there is something malformed about this component.
The `DashboardAvatar` should not be aware about its container. This component should have two variants : `square` or `rounded` then if there is some position customisation to apply when it is used into Sidebar context, this should be done from the sidebar component. WDYT ?

Reply:
Yes it's better to always have parent in charge of positioning the children.
Here it seems overkill to me to refactor the interaction between theses components, DashboardAvatar shapes and positions don't need to be deeply customize.
Also, i feel like the shape and position goes together, a circle is nicer center et a square a bit higher.

I'll change it to make the code more standard.